README.md

```diff
@@ -20,7 +20,8 @@ Running the tests
 -----------------
 
 ```
-docker-compose run --rm ci venv/bin/pytest
```

Comment:
The current command doesn't run locally as there is no `ci` service in `docker-compose.yml` or `docker-compose.ci.yml`. It's not great to re-build every time but it let me run tests in batched.

Reply:
Looks like we're missing a `cli` image that could be used in `dev`.